test/ao/sparsity/test_activation_sparsifier.py

```diff
@@ -57,6 +60,48 @@ def reduce_fn(x):
             'reduce_fn': reduce_fn
         }, sparse_config
 
+    @staticmethod
+    def _get_register_layer_args(model):
+        r"""Builds configuration for the layers that is going to be attached to the sparsifier.
+
+        Returns:
+            a list of tuples
+                Each entry in the list corresponds to the layer arguments.
+                First entry in the tuple corresponds to all the arguments other than sparse_config
+                Second entry in the tuple corresponds to sparse_config
+        """
+        def _vanilla_norm_sparsifier(data, sparsity_level):
+            r"""Similar to data norm spasifier but block_shape = (1,1).
+            Simply, flatten the data, sort it and mask out the values less than threshold
+            """
+            data_norm = torch.abs(data).flatten()
+            _, sorted_idx = torch.sort(data_norm)
+            threshold_idx = round(sparsity_level * len(sorted_idx))
+            sorted_idx = sorted_idx[:threshold_idx]
+
+            mask = torch.ones_like(data_norm)
+            mask.scatter_(dim=0, index=sorted_idx, value=0)
+            mask = mask.reshape(data.shape)
+
+            return mask
+
+        register_layer1_args = {
+            'layer': model.conv1,
+            'mask_fn': _vanilla_norm_sparsifier
+        }
+        sparse_config_layer1 = {'sparsity_level': 0.3}
+
+        register_layer2_args = {
+            'layer': model.linear1,
+            'features': [0, 10, 234],
+            'feature_dim': 1,
+            'mask_fn': _vanilla_norm_sparsifier
+        }
```

Comment:
If these are hardcoded, I am not sure we need a dedicated getter method for it? Can't we just use `_vanilla...` without the wrapping method?

Reply:
done